WIP-CHANGELOG.md

```diff
@@ -31,10 +31,11 @@
  - Upgrade velero helm chart to `v2.27.3`, which also upgrades velero to `v1.7.1`.
  - Upgrade prometheus-elasticsearch-exporter helm chart to v4.11.0 and prometheus-elasticsearch-exporter itself to v1.3.0
  - Exposed options for starboard-operator to control the number of jobs it generates and to allow for it to be disabled.
-
+ - Thanos is now enabled by default
```

Comment:
```suggestion
 - Thanos is now enabled by default
 
```

Reply:
This does not look like it is fixed. There is still no empty line between this like and `### Fixed`.